app/controllers/hotels_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HotelsController < ApplicationController
+  def show
+    @hotel = Hotel.find(params[:id])
+    @unique_guests = @hotel.unique_guests
```

Comment:
This is fine but you can also keep your controller "skinny" by calling `.unique_guests` in your Show view.